source/lac/dynamic_sparsity_pattern.cc

```diff
@@ -512,6 +540,13 @@ template void DynamicSparsityPattern::Line::add_entries(size_type *,
 template void DynamicSparsityPattern::Line::add_entries(const size_type *,
                                                         const size_type *,
                                                         const bool);
+
+template void DynamicSparsityPattern::compute_mmult_pattern(
+  DynamicSparsityPattern &,
+  const DynamicSparsityPattern &) const;
+template void DynamicSparsityPattern::compute_mmult_pattern(
+  DynamicSparsityPattern &,
+  const SparsityPattern &) const;
```

Comment:
Maybe move this below the `add_entries` function to keep all explicit instantiations of `add_entries` together.